acorn.js

```diff
@@ -2393,6 +2393,7 @@
     classBody.body = [];
     expect(_braceL);
     while (!eat(_braceR)) {
+      if (eat(_semi)) continue;
```

Comment:
It will eat all semicolons without having to check for right brace every character.